### PR TITLE
add optional service annotations to nexus chart

### PR DESCRIPTION
**Pull request by rawlingsj**

### CODE REVIEW

File: stable/sonatype-nexus/values.yaml [hunk 1]
```diff
@@ -17,6 +17,8 @@ service:
   livenessProbe:
     initialDelaySeconds: 30
     periodSeconds: 30
+  annotations: {}
```
Comment (mattfarina) [inline stable/sonatype-nexus/values.yaml]: You can remove the `{}` from the end of the line. For example, you can see the redmine chart

https://github.com/kubernetes/charts/blob/78461d1ab08a2a815a61cb566a8130bd5535ef37/stable/redmine/values.yaml#L124-L129

**Review by mattfarina — COMMENTED** [review]
You also need to increment the chart version for this to pass CI.

**Comment by rawlingsj** [comment]
Thanks @mattfarina, commit updated with chart version incremented and unnecessary `{}` removed.

**Pull request merged**